test/e2e/egress_test.go

```diff
@@ -28,15 +28,14 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apimachinery/pkg/util/wait"
+	utilnet "k8s.io/utils/net"
 
 	"antrea.io/antrea/pkg/agent/config"
 	"antrea.io/antrea/pkg/apis/crd/v1alpha2"
 )
 
 func TestEgress(t *testing.T) {
 	skipIfProviderIs(t, "kind", "pkt_mark field is not properly supported for OVS userspace (netdev) datapath.")
-	// TODO: remove this after making the test support IPv6 and dual-stack.
-	skipIfIPv6Cluster(t)
```

Comment:
I suggest adding `skipIfNumNodesLessThan(t, 2)` as well since most egress tests seem to require at least 2 Nodes

Reply:
Done